doc/manual/macros

```diff
@@ -72,6 +72,7 @@ to perform useful operations. The current list is
 	%define ...	define a macro
 	%undefine ...	undefine a macro
 	%global ...	define a macro whose body is available in global context
+	%{body:...}	literal body of a macro
```

Comment:
I am sorry, but from the macro name neither from its description I can tell what it does. Isn't 'verbatim' more fitting name here?

Reply:
See the discussion in #582, the problem with %{verbatim:..} and other similar variants is that the argument is ambiguous, whereas the term "body" is unambiguous within rpm terminology, including the document in question. The description could be better, no disagreement there.